rl_chain/rl_chain_base.py

```diff
@@ -339,7 +341,7 @@ def _call(
         self.policy.learn(event=event)
         self.policy.log(event=event)
 
-        return {self.output_key: {"response": output, "decision_metadata": event}}
+        return {self.output_key: {"response": output, "selection_metadata": event}}
```

Comment:
actually why do we need outer "output_key" dictionary?

Reply:
the chain uses it, you can only set one and it removes it before returning the object so you only get the dictionary on return